src/app/modules/item/pages/item-log-view/item-log-view.component.ts

```diff
@@ -26,6 +26,7 @@ export class ItemLogViewComponent implements OnChanges, OnDestroy {
 
   @Input() itemData?: ItemData;
   @Input() isWatchingGroup = false;
+  @Input() watchingGroupId?: string;
```

Comment:
Get this information directly from the session service as we did in the other places. (it is not a pure UI component (like a picker or a button) anyway).

Reply:
Fixed